docs/Release-Process.md

```diff
@@ -73,31 +69,38 @@ The process should go something like:
 Please keep the local replace references for `cloud-providers`, `peerpod-ctrl` and `cloud-api-adaptor`
 and run `make tidy` under the [cloud-api-adaptor](../) to update packages for each go modules.
 
-- Merge the PR with this update to update the `main` branch
+- Create and merge the PR with the operator version and go module commits to update the `main` branch. When this change
+is merged,it triggers the [project images publish workflow](../.github/workflows/publish_images_on_push.yaml) to
+create a new container image in
+[`quay.io/confidential-containers/cloud-api-adaptor`](https://quay.io/repository/confidential-containers/cloud-api-adaptor?tab=tags)
+to use in testing.
 
-- Create git tags for all go modules, you can use the [release-helper.sh](../hack/release-helper.sh) script to create related git commands, (e.g. `v0.8.0-alpha.1`)
+- Create git tags for all go modules, you can use the [release-helper.sh](../hack/release-helper.sh) script with the `go-tag` command
+to create related git commands, (e.g. `v0.8.0-alpha.1`)
+> **Note:** `hack/release-helper.sh` `go-tag` has an optional third parameter for the name of the upstream remote,
+which defaults to origin if not supplied
 ```bash
-./hack/release-helper.sh v0.8.0-alpha.1
+./hack/release-helper.sh go_tag v0.8.0-alpha.1 upstream
```

Comment:
Hi @stevenhorsman !

I missed the PR that introduced the `release-helper.sh`. Great script!

One minor fix here, the command is `go-tag` rather than `go_tag`.

Reply:
Everything else looks good. I will give my approval upfront ;)

Reply:
Ah sorry, I started with `go_tag` and then renamed it, but obviously missed one. Thanks for spotting.